package.json

```diff
@@ -13,77 +51,39 @@
     "svg",
     "canvas"
   ],
-  "author": "Gabe Lerner <gabelerner@gmail.com>",
-  "contributors": [
-    "Brett Zamir"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/canvg/canvg.git"
-  },
-  "main": "dist/node/canvg.js",
-  "types": "dist/node/canvg.d.ts",
-  "browser": "dist/browser/canvg.min.js",
-  "version": "2.0.0",
-  "scripts": {
-    "test-browser": "ava test/browser.test.js --tap | tap-diff",
-    "test-node": "ava test/node.test.js --tap | tap-diff",
-    "test-common": "ava test/common.test.js --tap | tap-diff",
-    "test": "npm run test-common && npm run test-node",
-    "minify-browser": "uglifyjs --mangle --compress if_return=true -o dist/browser/canvg.min.js dist/browser/canvg.js",
-    "minify": "npm run minify-browser",
-    "eslint": "eslint --report-unused-disable-directives .",
-    "build-browser": "touch src/dummy.js && rollup -c && rm src/dummy.js",
-    "build-node": "IS_NODE=1 rollup -c",
-    "build-win": "npm run build-browser-win && npm run build-node-win && npm run minify && del src\\dummy.js",
-    "build-browser-win": "type nul > src\\dummy.js && rollup -c",
-    "build-node-win": "cmd /C \"set IS_NODE=1 && rollup -c\"",
-    "build": "npm run eslint && npm run build-browser && npm run build-node && npm run minify",
-    "generate-expected": "node test/_generate_expected_node.js",
-    "start": "node test/_server.js"
-  },
-  "license": "MIT",
-  "files": [
-    "dist",
-    "LICENSE",
-    "README.md",
-    "package.json"
-  ],
-  "readmeFilename": "README.md",
   "dependencies": {
+    "@babel/runtime": "^7.6.3",
+    "@types/raf": "^3.4.0",
+    "core-js": "3",
+    "raf": "^3.4.1",
     "rgbcolor": "^1.0.1",
     "stackblur-canvas": "^2.0.0"
   },
-  "peerDependencies": {
-    "canvas": "^2.1.0",
-    "jsdom": "^13.0.0",
-    "xmldom": "^0.1.27"
-  },
   "devDependencies": {
-    "ava": "^1.0.0-rc.2",
-    "ava-is-running": "^1.0.3",
-    "blink-diff": "^1.0.13",
-    "bluebird": "^3.5.3",
+    "@size-limit/preset-small-lib": "^2.1.6",
+    "@trigen/scripts": "3.2.0",
+    "@trigen/scripts-plugin-babel": "3.2.0",
+    "@trigen/scripts-plugin-eslint": "3.2.0",
+    "@trigen/scripts-plugin-jest": "3.2.0",
+    "@trigen/scripts-plugin-rollup": "3.1.0",
+    "@trigen/scripts-plugin-size-limit": "3.2.0",
+    "@trigen/scripts-plugin-typescript": "3.2.0",
+    "@trigen/scripts-preset-lib": "3.2.0",
```

Comment:
I don't know any "more popular versions".

Reply:
i looked through some of them and most are just wrappers around other scripts with some default args. can we just take what's needed and put it into the `package.json` directly or worst case create a `script.sh` or `script.js` here? these foreign packages are really the only dealbreaker for me in the whole PR - the rest is amazing! what I meant by "popular" is that there are plenty of TS/JS projects that get by w/o the overhead and risk of having these kinds of packages as dependencies. we shouldn't need `trigen-scripts` to call `jest` or `lint` ...

Reply:
since you did dedicate a lot of time to this, i'm more than happy to put a `thanks to trigen software` link in the readme - i just don't feel we need cohesion between the two repos where sometimes we'll be waiting for changes to trigen just to deploy canvg. as another alternate option, we can duplicate all the scripts to canvg's org and publish to npm from here.